### DISCUSSION TITLE
Searching for an issue (eg. By number) is fruitless

### DISCUSSION THREAD

**Discussion by SoylentGraham** [community/community#108368] [Mobile]
<details>
I have an issue number in one of my or my org's repositories;
I have no direct link (so no deep link).
I search issues for 1234

I get results from hundreds of global repositories... but none are mine, and none seem to be #1234

I cant even pick an org or repos to filter by.

Are people really using the issue search to search for text globally? :) 

<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.148.0 (122177434)
Device: iPhone SE (1st generation) (iOS 15.7.8)
</details>

**Comment by HenSquared** [comment]
Hi, thanks for the feedback. Instead of using global search, can you try searching for your issue number using 'Issues' screen from Home > My Work? 

Does that help resolve the issue? Thanks!

**Reply by SoylentGraham** [reply]
This doesn't really help this case, as although I'm in the org, I'm not "involved" (creator, mentioned, etc) so it doesn't come up in the search.
But I really want to see it, and I know exactly what I'm looking for, and there's only one issue in all my repositories with this number :)

**Reply by SoylentGraham** [reply]
I basically, just cannot access this issue without a full url to deep-link via

**Reply by Iron-Ham** [reply]
@SoylentGraham if you know the issue number, repo-name, and repo-owner, there is a handy shortcut within the search view.

For example, let's say I wanted to get to: https://github.com/apollographql/apollo-ios/issues/3333

In the search bar, I could type: `apollographql/apollo-ios#3333`. That will allow me to directly jump to that issue. 

See:

https://github.com/community/community/assets/3388381/6ce49e6d-8e1d-4798-a4b2-fba3a6fb2516

This feature is available on the iPhone's Home search, the iPad's search tab, and both device's Explore tab.

**Reply by SoylentGraham** [reply]
I think you may be stretching the definition of "shortcut" ;)
